requirements.txt

```diff
@@ -217,19 +235,23 @@ pyasn1-modules==0.4.0
     # via google-auth
 pycparser==2.22
     # via cffi
-pydantic==2.7.4
+pydantic==2.8.2
     # via
     #   fastapi
     #   pydantic-extra-types
     #   pydantic-settings
-pydantic-core==2.18.4
+pydantic-core==2.20.1
     # via pydantic
-pydantic-extra-types==2.8.2
+pydantic-extra-types==2.9.0
     # via fastapi
-pydantic-settings==2.3.4
+pydantic-settings==2.4.0
     # via fastapi
+pygments==2.18.0
+    # via rich
 pymysql==1.1.1
```

Comment:
I'm a little confused as to why pip-audit complained about 1.1.0 when we already had 1.1.1 installed?

Reply:
I believe the scan we were looking at was from `requirements-dev.txt` only, which has it at `1.1.0`. So maybe I should only pin this for dev instead

Reply:
Ah yep, that makes sense, would be cool if pip-audit could make clear whether it is a dev dependency or prod dependency that has the vulnerability

Reply:
That would be nice indeeeeeed